lib/active_admin/middleware/error_handler.rb

```diff
@@ -0,0 +1,26 @@
+module ActiveAdmin
+  class ErrorHandler
+
+    def initialize(app)
+      @app = app
+    end
+
+    def call(env)
+      @app.call(env)
+    rescue Exception => e
+      if ActiveAdmin.application.default_namespace.to_s == env['PATH_INFO'].split('/').second
+        render_exception(env, e)
+      else
+        @app.call(env)
+      end
+    end
```

Comment:
Registering an error page and using it to show error is not allowed in Activeadmin.

I tried it. I am seeing an exception raised by the base controller, that `AbstractController::ActionNotFound`

https://github.com/activeadmin/activeadmin/blob/master/lib/active_admin/base_controller.rb#L28

This is the reason why I went with using ActionController for the exception controller.

My first choice was to use the base controller, so that I would need not have to implement all the helpers provided by it. But because of the exception, I did not continue with it.